src/wp-includes/option.php

```diff
@@ -258,14 +258,24 @@ function get_option( $option, $default_value = false ) {
  *
  * @param array $options An array of option names to be loaded.
  */
-function wp_load_options( $options ) {
+function wp_prime_option_caches( $options ) {
 	$alloptions     = wp_load_alloptions();
-	$cached_options = wp_cache_get_multiple( $options, 'options' );
+	$cache_keys     = (array) $options;
+	$cache_keys[]   = 'notoptions';
+	$cached_options = wp_cache_get_multiple( $cache_keys, 'options' );
+
+	// Store notoptions, remove from cached options.
+	$notoptions = $cached_options['notoptions'];
```

Comment:
Note that `$notoptions` is also retrieved further down in the function. Maybe we can remove that line now if we already set it up here?